refactor: Update utility function import paths

ilhanbae opened the pull request:
# Task

* I removed `utils/index.ts`
* I updated utility function import paths

[Link to Related Ticket](https://incredihire-academy.atlassian.net/jira/software/c/projects/SR/boards/3?selectedIssue=SR-104)
[Link to Related PR](https://github.com/Incredihire/Survey/pull/89)

## Acceptance Criteria
- [x] N/A

Sergei-S-V reviewed (commented):
Personally, I find barrel files quite useful as they help keep imports shorter and cleaner. They also make refactoring easier - when files are moved within a directory (like utils), you only need to update the index.ts instead of changing all the import paths across the project.
It's a matter of preference, but they're widely used in most projects.

ilhanbae commented:
@Sergei-S-V Sorry for not addressing your comment, I had auto-merge enabled. Having barrel file is definitely useful - it keeps imports shorter and cleaner as you mentioned. The reason for this change is that some files were importing the util functions from modules and some were importing from barrel file, and I wanted our codebase to be consistent. @jakethrasher also mentioned in my previous PR that there's a possibility negative effect on tree shaking.
